packages/server/src/utils.js

```diff
@@ -60,10 +60,13 @@ export let runTypeFunction = (config) => async (name, node, search) => {
     return await (search
       ? fn(node, search, schema, config)
       : fn(node, schema, config))
-  } catch (error) {
+  } catch (e) {
     // Sometimes we throw strings instead of errors
-    const message = _.getOr(error, 'message', error)
-    error.message = `(at contexture node with key=${node.key} and type=${node.type}) ${message}`
+    const error = _.isString(e) ? new Error(e) : e
+    error.cause = {
+      contexture: { node },
+      ...(_.isString(error.cause) ? { error: error.cause } : error.cause ?? {}),
```

Comment:
This line causes an error because the precedence of ?? is higher the ?, replace ?? with || to fix this.

[Here is a reference to this precedence in documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing#:~:text=The%20nullish%20coalescing%20operator,thrown%20in%20such%20cases.).

Reply:
Did you test this and found an error?

The intention is to always get an object so the spread doesn't fail, so `x ? {} : z ?? {}` and `(x ? {} : z) ?? {}` are equivalent.

Reply:
the nullish operator has a higher precedence, so there is an error in the code when testing this in isolation on runkit. It produces an error when attempting to run because of this, you cannot combine ?? and ? apparently because of the precedence issue. Even with they simple example you posted it still has an error.

Reply:
Actually, I think that runkit has an issue with ?? which makes no sense because the versions look up to date.